keyboards/sofle/keymaps/via/config.h

```diff
@@ -21,6 +21,7 @@
 see https://docs.qmk.fm/#/feature_split_keyboard?id=setting-handedness
 for more options.
 */
+#define MASTER_LEFT
```

Comment:
This isn't needed.  If no other handedness define is present, master left is the default behavior. 
```suggestion
```

Reply:
Sure, i've revised the comment then, as it seemed like there is something missing.